src/zos_console/zowe/zos_console_for_zowe_sdk/response/__init__.py

```diff
@@ -10,4 +10,7 @@
 Copyright Contributors to the Zowe Project.
 """
 
-from .console import ConsoleResponse, IssueCommandResponse
+from .console_response import ConsoleResponse 
+from .issue_response import IssueCommandResponse
+
+__all__= ["ConsoleResponse", "IssueCommandResponse"]
```

Comment:
I won't claim to be an expert on how this all works, but where is `.issue_response`?
I'm afraid I can't get the unit tests to run because of this (and many similar changes) 🤔

Reply:
See newer comments 😋